### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by AVyha**

### CODE REVIEW

**Review by olia-trofymchuk — APPROVED** [review]
As for me, your solution is good, but maybe for best practices you should change one-quotes in "" :))

File: app/urls.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path, include
+
+urlpatterns = [
+    path('admin/', admin.site.urls),
+    path("api/cinema/", include("cinema.urls", namespace="cinema"))
```
Comment (TPolina) [inline app/urls.py:21]: don't mix styles of quotes. Fix it everywhere

File: cinema/views.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=200)
```
Comment (TPolina) [inline cinema/views.py:15]: use status from rest_framework. Fix everywhere

File: cinema/views.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies_list(request):
```
Comment (TPolina) [inline cinema/views.py:11]: `movie_list` is better

File: cinema/views.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=201)
+
+        return Response(serializer.errors, status=400)
+
+
+@api_view(["GET", "PUT", "DELETE"])
+def movies_object(request, pk: int):
```
Comment (TPolina) [inline cinema/views.py:28]: `movie_detail` is better

**Review by TPolina — CHANGES_REQUESTED** [review]

**Review by TPolina — APPROVED** [review]

**Pull request closed without merging**